README.md

```diff
@@ -62,7 +62,6 @@ By default, Octotree only works on `github.com`. To support enterprise version (
 
```

Comment:
Please update the screenshots.

Reply:
I don't have a big screen computer. Could someone help me with taking the screenshot ?

Reply:
test

Reply:
test

Reply:
new